internal/yunikorn/event_handler.go

```diff
@@ -89,12 +89,21 @@ func (s *Service) handleQueueEvent(ctx context.Context, ev *si.EventRecord) {
 		return
 	}
 
+	// Sync partitions before syncing queues
+	partitions, err := s.client.GetPartitions(ctx)
```

Comment:
We should check if partition exists. If not, only then we should sync

Reply:
What if the partition exists in our database, but some of its data has changed on the YuniKorn side? If we only sync when the partition doesn’t exist in our database,  we will end up with the initial state of the partition always.